gensim/models/tfidfmodel.py

```diff
@@ -273,21 +282,33 @@ def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.iden
                 * `c` - cosine.
 
             For more information visit [1]_.
-
+        slope : float, optional
+            It is the parameter required by pivoted document length normalization which determines the slope to which
+            the `old normalization` can be tilted.
+        pivot : float, optional
+            It is the point around which the traditional normalization curve is `tilted` to get the new pivoted
```

Comment:
Need more detailed description (previous was better, but not complete, right now - this worse that was). Also, you can add link to original paper here too.

Also, what happens if `pivot` is None (or not None), this didn't mention here.